atom-feed/pom.xml

```diff
@@ -84,7 +84,7 @@
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
                                             <!-- Lowered as PowerMockito is not covered -->
-                                            <minimum>0.43</minimum>
+                                            <minimum>0.37</minimum>
```

Comment:
any reason we are downgrading the version?

Reply:
https://bahmni.atlassian.net/browse/BAH-1926
Created a ticket for analysis of this.